incubator/README.md

```diff
@@ -0,0 +1,33 @@
+# Incubator Registry
+
+## Overview
+
+This directory is an official ksonnet-compatible [registry][2]. If you are unfamiliar with ksonnet, we recommend browsing [the official site][1] to gain more context.
+
+ksonnet's CLI tool (`ks`) is aware of the `incubator` registry out-of-the-box.
```

Comment:
Minor nit but consider (and feel free to reject) changing this to:
Out-of-the-box, ksonnet's CLI tool ('ks') is aware of the 'incubator' registry